src/main/kotlin/dev/deftu/omnicore/api/client/options/GuiScale.kt

```diff
@@ -1,16 +1,15 @@
-package dev.deftu.omnicore.client.render
+package dev.deftu.omnicore.api.client.options
 
-import dev.deftu.omnicore.annotations.GameSide
-import dev.deftu.omnicore.annotations.Side
-import dev.deftu.omnicore.client.OmniClient
+//#if MC >= 1.16.5 && MC < 1.20.5
+//$$ import dev.deftu.omnicore.api.client.client
+//#endif
 
 /**
  * Represents the GUI scales Minecraft provides, which force render operations to be scaled by a certain factor.
  *
  * @since 0.2.1
  * @author Deftu
  */
-@GameSide(Side.CLIENT)
 public enum class GuiScale {
```

Comment:
On aother note, it's possible to have gui scales up to 8 in modern mc so this makes no sense